src/pages/Dashboard/Dashboard.js

```diff
@@ -39,7 +39,7 @@ export default function Dashboard ({ earnings = mockEarnings }) {
     </div>
     <h2 styleName='greeting'>{greeting}</h2>
 
-    <Card title='Hosting' linkTo='/browse-happs' subtitle='Manage your Holo applications'>
+    {false && <Card title='Hosting' linkTo='/browse-happs' subtitle='Manage your Holo applications'>
```

Comment:
I would normally avoid this, but we know we're going to be introducing these cards back in exactly as is very soon, so I made an exception.